src/Knp/Component/Pager/PaginatorInterface.php

```diff
@@ -21,7 +21,9 @@ interface PaginatorInterface
     public const FILTER_VALUE_PARAMETER_NAME = 'filterValueParameterName';
     public const FILTER_FIELD_WHITELIST = 'filterFieldWhitelist';
     public const DISTINCT = 'distinct';
-
+    public const FORCE_VALID_PAGINATION_PARAMETERS = 'forceValidPaginationParameters';
+    public const DEFAULT_LIMIT = 'defaultLimit';
+    public const DEFAULT_PAGE_NUMBER = 'defaultPageNumber';
```

Comment:
@daria-sieroshtan 
I suggest to remove last two items
Also please return line break before the method